### PR TITLE
Update README to v0.2.0

### PR DESCRIPTION
**Pull request by guangbochen**
Update README & docs to reflect v0.2.0 changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,16 +10,17 @@ Rancher Harvester is an open source [hyper-converged infrastructure](https://en.
 ## Overview
 Harvester implements HCI on bare metal servers. Here are some notable features of the Harvester:
 1. VM lifecycle management including SSH-Key injection, Cloud-init and, graphic and serial port console
+1. VM backup/restore and live migration support
```
Comment (gitlawr) [inline README.md]: Seems to be two items

File: README.md [hunk 2]
```diff
@@ -42,33 +43,41 @@ Users can use the ISO to install Harvester directly on the bare-metal server to
...
 1. Configure the `cluster token`. This token will be used for adding other nodes to the cluster.
 1. Configure the login password of the host. The default ssh user is `rancher`.
 1. (Optional) you can choose to import SSH keys from a remote URL server. Your GitHub public keys can be used with `https://github.com/<username>.keys`.
-1. Select the network interface for the management network.
 1. (Optional) If you need to use an HTTP proxy to access the outside world, enter the proxy URL address here, otherwise, leave this blank.
 1. (Optional) If you need to customize the host with cloud-init config, enter the HTTP URL here.
 1. Confirm the installation options and the Harvester will be installed to your host. The installation may take a few minutes to be complete.
 1. Once the installation is complete it will restart the host and a console UI with management URL and status will be displayed. <small>(You can Use F12 to switch between Harvester console and the Shell)</small>
-1. The default credentials for the web interface are username:`admin` and password: `password`.
+1. The default URL of the web interface is `https://your-network-ip:30443`.
```
Comment (gitlawr) [inline README.md]: s/your-network-ip/your-host-ip/

File: README.md [hunk 3]
```diff
@@ -42,33 +43,41 @@ Users can use the ISO to install Harvester directly on the bare-metal server to
...
 ![iso-installed.png](./docs/assets/iso-installed.png)
+1. User will be prompted to set the root password for the default `admin` user on the first-time login.
```
Comment (gitlawr) [inline README.md]: s/root password/password/

File: docs/README.md [hunk 4]
```diff
@@ -7,8 +7,8 @@ or hang out with our developers on either our [Slack](https://slack.rancher.io/)
 
 - Installation
   * [ISO Mode](./iso-installation.md)
-  * [App Mode - for development](./app-mode-installation.md)
   * [Automatic Mode - for mass installation](./automatic-installation.md)
+  * [App Mode - for development](./app-mode-installation.md)
```
Comment (gitlawr) [inline docs/README.md]: We removed the App mode and call it Dev mode recently(https://github.com/harvester/harvester/commit/3758022490af00442191ca3c58a8c911a4bf12ef).
We may also want to update that in Github issue templates

File: docs/authentication.md [hunk 5]
```diff
@@ -1,14 +1,23 @@
 # Authentication
-Harvester Dashboard supports `local auth` mode for authentication. The default username and password is `admin/password`.
+
+>  Available as of v0.2.0
+
+With ISO installation mode, user will be prompted to set the root password for the default `admin` user on the first-time login.
```
Comment (gitlawr) [inline docs/authentication.md]: s/root password/password/

File: docs/authentication.md [hunk 6]
```diff
@@ -1,14 +1,23 @@
...
 ## App Mode
 In `App mode`, which is intended only for development and testing purposes, more authentication modes are configurable using the environment variable `HARVESTER_AUTHENTICATION_MODE`.
 
+Harvester Dashboard default to use `local auth` mode for authentication. The default username and password is `admin/password`.
+
 The currently supported options are `localUser` (the same as `local auth` mode) and `kubernetesCredentials`.
 
-If the `kubernetesCredentials` authentication option is used, either a kubconfig file or bearer token can provide access to Harvester.
\ No newline at end of file
+If the `kubernetesCredentials` authentication option is used, either a `KUBCONFIG` file or bearer token can provide access to Harvester.
```
Comment (gitlawr) [inline docs/authentication.md]: We don't need to capitalize it. Kubeconfig is a common word(https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/). It was a typo missing e though.

File: docs/create-vm.md [hunk 7]
```diff
@@ -14,6 +14,31 @@ Create one or more virtual machines from the **Virtual Machines** page.
 
 ![](./assets/create-vm.png)
 
+### Cloud-init config examples
+
+- set root password of a cloud image
+```YAML
+#cloud-config
+password: password
```
Comment (gitlawr) [inline docs/create-vm.md]: I think this is for the default user, not root

File: docs/create-vm.md [hunk 8]
```diff
@@ -24,4 +49,4 @@ By default, a VM can be accessed via the management network.
 
 ##### Secondary Network
 
-It is also possible to connect VMs using the secondary networks with build-in vlan networks. This is powered by [Multus](https://github.com/intel/multus-cni). 
\ No newline at end of file
+It is also possible to connect VMs using the additional networks with Harvester's built-in [vlan networks](./harvester-network.md). This is powered by [Multus](https://github.com/intel/multus-cni).
```
Comment (gitlawr) [inline docs/create-vm.md]: s/the additional/additional/

File: docs/iso-installation.md [hunk 9]
```diff
@@ -2,20 +2,24 @@
...
+![iso-installed.png](./assets/iso-nic-config.png)
 1. Configure the `cluster token`. This token will be used for adding other nodes to the cluster.
 1. Configure the login password of the host. The default ssh user is `rancher`.
 1. (Optional) you can choose to import SSH keys from a remote URL server. Your GitHub public keys can be used with `https://github.com/<username>.keys`.
-1. Select the network interface for the management network.
 1. (Optional) If you need to use an HTTP proxy to access the outside world, enter the proxy URL address here, otherwise, leave this blank.
 1. (Optional) If you need to customize the host with cloud-init config, enter the HTTP URL here.
 1. Confirm the installation options and the Harvester will be installed to your host. The installation may take a few minutes to be complete.
 1. Once the installation is complete it will restart the host and a console UI with management URL and status will be displayed. <small>(You can Use F12 to switch between Harvester console and the Shell)</small>
+1. The default URL of the web interface is `https://your-network-ip:30443`.
```
Comment (gitlawr) [inline docs/iso-installation.md]: ditto

File: docs/iso-installation.md [hunk 10]
```diff
@@ -2,20 +2,24 @@
...
 ![iso-installed.png](./assets/iso-installed.png)
+1. User will be prompted to set the root password for the default `admin` user on the first-time login.
```
Comment (gitlawr) [inline docs/iso-installation.md]: ditto

File: docs/rancher-integration.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+
+1. Click the actions of the `rancher-enabled` setting.
+1. Select the `Enable` option and click the save button.
+1. On the top-right corner the Rancher dashboard button will appear.
+1. Click the Rancher button, and it will open a new tab to navigate to the Rancher dashboard.
+1. For more detail about how to use the Rancher you may refer to this [doc](https://rancher.com/docs/rancher/v2.5/en/).
+
+
+
+# Creating k8s Cluster using the Harvester Node Driver
```
Comment (gitlawr) [inline docs/rancher-integration.md]: clusters

File: docs/upload-image.md [hunk 12]
```diff
@@ -4,6 +4,8 @@ To upload virtual machine images in the **Images** page, enter a URL that can be
 
 Currently, we support qcow2, raw, and ISO images.
 
+> Note: uploading images from the local desktop to the Harvester cluster is not supported yet, will be tracked on [#570](https://github.com/harvester/harvester/issues/570).
```
Comment (gitlawr) [inline docs/upload-image.md:7]: Should we change the term `upload` in this file?

Comment (guangbochen) [reply docs/upload-image.md:7]: probably leave it as what it is since this is a not-supported feature.

**Comment by guangbochen** [comment]
@gitlawr updated, and I will just cancel the CI since is not needed

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**